main.go

```diff
@@ -48,6 +48,10 @@ type config struct {
 	// External URI to /api
 	APIPrefix string `yaml:"api_prefix"`
 
+	// Allowed rageshake app names
+	AllowedAppNames   []string `yaml:"allowed_app_names"`
+	AllowedAppNameMap map[string]bool
```

Comment:
this probably doesn't need exporting given it is just a memoized helper to quicker lookups